tests/e2e/specs/in-game/sevens.spec.js

```diff
@@ -820,22 +826,18 @@ describe('Opponent playing SEVENS', () => {
     // Deck cards appear but are not selectable
     cy.get('[data-top-card=4-0]')
       .should('exist')
-      .and('be.visible')
       .click({ force: true })
       .should('not.have.class', 'selected');
     cy.get('[data-second-card=6-1]')
       .should('exist')
-      .and('be.visible')
       .click({ force: true })
       .should('not.have.class', 'selected');
-    cy.get('#scrap').should('be.visible').and('not.have.class', 'valid-move').click({ force: true }); // can't play to scrap
-    cy.get('#player-field').should('not.have.class', 'valid-move').click({ force: true }); // can't play to field
```

Comment:
These were out of date anyway; clicking the scrap isn't how you play one-offs anymore and clicking the field isn't how you play points.